src/Model/Email/HoldEmail.php

```diff
@@ -26,9 +27,12 @@ public function getFromAddress()
      */
     public function getToAddress()
```

Comment:
There are a couple of minor syntax issues, but overall it looks fine to me. You may want to set up a Patron class for patron specific data/manipulations and what not as mentioned above.